<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotView</class>
 <widget class="QFrame" name="PlotView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="contentParent" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="multiToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../ocurves.qrc">
         <normaloff>:/icons/multi-tool.png</normaloff>:/icons/multi-tool.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="zoomToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../ocurves.qrc">
         <normaloff>:/icons/rubber-band.png</normaloff>:/icons/rubber-band.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="panToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../ocurves.qrc">
         <normaloff>:/icons/move.png</normaloff>:/icons/move.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Legend:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="legendCombo">
       <property name="currentText">
        <string>Left</string>
       </property>
       <item>
        <property name="text">
         <string>Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bottom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Top</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shared</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="panLockButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../ocurves.qrc">
         <normaloff>:/icons/unlock.png</normaloff>
         <normalon>:/icons/lock.png</normalon>:/icons/unlock.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="zoomXButton">
       <property name="text">
        <string>X</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="zoomYButton">
       <property name="text">
        <string>Y</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="zoomBothButton">
       <property name="text">
        <string>X/Y</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionMultiTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ocurves.qrc">
     <normaloff>:/icons/multi-tool.png</normaloff>:/icons/multi-tool.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Multi Tool</string>
   </property>
  </action>
  <action name="actionZoomTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ocurves.qrc">
     <normaloff>:/icons/rubber-band.png</normaloff>:/icons/rubber-band.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom Tool</string>
   </property>
   <property name="toolTip">
    <string>Zoom tool</string>
   </property>
  </action>
  <action name="actionPanTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ocurves.qrc">
     <normaloff>:/icons/move.png</normaloff>:/icons/move.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pan Tool</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../ocurves.qrc"/>
 </resources>
 <connections/>
</ui>
